interface/wx/filepicker.h

```diff
@@ -351,23 +409,36 @@ class wxDirPickerCtrl : public wxPickerBase
 class wxFileDirPickerEvent : public wxCommandEvent
 {
 public:
+
+    /**
+        Default constructor.
+    */
     wxFileDirPickerEvent();
 
     /**
-        The constructor is not normally used by the user code.
+        Constructor with picker related path string.
+
+        @param type
+            Control specific event types. Can be @c wxEVT_FILEPICKER_CHANGED or 
+            @c wxEVT_DIRPICKER_CHANGED.
+        @param generator
+            Object creating the event.
+        @param id
+            wxWindowID of the window creating the event.
+        @param path
+            Newly selected absolute path to the file or directory.
     */
     wxFileDirPickerEvent(wxEventType type, wxObject* generator,
                          int id,
                          const wxString& path);
 
     /**
-        Retrieve the absolute path of the file/directory the user has just selected.
+        Retrieve the new absolute path of the file/directory.
```

Comment:
I'm not sure about the real benefit of a few changes done here, but this one doesn't seem like an improvement at all to me, the old wording was more clear IMHO.